.github/workflows/ci.yaml

```
@@ -36,7 +36,7 @@ jobs:
 
       - uses: ruby/setup-ruby@v1
         with:
-          ruby-version: ${{ matrix.ruby }}
+          ruby-version: ${{ matrix.ruby.value }}
           bundler: none
```

From what I understand, we don't need to install bundler manually and can either specify the version here so this would become:
```suggestion
          bundler: "2.2.15"
```
but also since we have `Gemfile.lock` files, the action by default will use what is specified in the `BUNDLED WITH` section as the bundler version so the bundler key here could be omitted entirely. https://github.com/ruby/setup-ruby#bundler